src/reynir/Greynir.grammar

```diff
@@ -1608,11 +1617,14 @@ OgSögn_mm_sagnb →
 
 OgSögn_sagnb/tala/kyn →
     # (hefur staðið) keikur í brúnni og mótmælt yfirgangi Rússa
-    LoTengtSögn_nf/tala/kyn? FsAtv? Aðaltenging FsAtv? EinSögn_sagnb/tala/kyn
+    LoTengtSögn_nf/tala/kyn? FsAtv? ÁframOgSögn_sagnb/tala/kyn* Aðaltenging FsAtv? EinSögn_sagnb/tala/kyn
     | LoTengtSögn_nf/tala/kyn? FsAtv? "," Aðaltenging FsAtv? EinSögn_sagnb/tala/kyn FsAtvKomma?
 
 $score(+6) OgSögn_sagnb/tala/kyn # Enn sterkari plús fyrir runu sagnb
 
+ÁframOgSögn_sagnb/tala/kyn →
+    "," EinSögn_sagnb/tala/kyn LoTengtSögn_nf/tala/kyn? FsAtv?
```

Comment:
Is the comma mandatory or optional? My gut feel is that it is optional and thus should be `","?`